jdaviz/tests/test_show.py

```diff
@@ -1,17 +1,11 @@
-import sidecar
+import pytest
 
 
-def test_show_inline(specviz_helper):
-    res = specviz_helper.show_inline()
+def test_show_popout(specviz_helper):
```

Comment:
Does this test actually exposes a bug here? I thought you wanted "popout" to emit a `NotImplementedError`.